# StairStep.msg
geometry_msgs/Pose pose
float64 step_width
float64 step_height
float64 step_length
bool is_ascent
